vm/src/anystr.rs

```diff
@@ -194,7 +194,7 @@ pub trait AnyStr {
         SW: Fn(&Self, isize, &VirtualMachine) -> Vec<R>,
     {
         let (sep, maxsplit) = args.get_value(vm)?;
-        let splited = if let Some(pattern) = sep {
+        let splitted = if let Some(pattern) = sep {
```

Comment:
"splitted" is nonstandard. Here, I'd recommend using `split_` (with a trailing underscore) or `the_split`, since `split` is already used.